axum/src/typed_header.rs

```diff
@@ -31,7 +31,24 @@ use std::ops::Deref;
 /// # axum::Server::bind(&"".parse().unwrap()).serve(app.into_make_service()).await.unwrap();
 /// # };
 /// ```
-#[cfg_attr(docsrs, doc(cfg(feature = "headers")))]
+///
+/// # As response
+///
+/// ```rust
+/// use axum::{
+///     TypedHeader,
+///     response::IntoResponse,
+///     headers::ContentType,
+/// };
+///
+/// async fn handler() -> impl IntoResponse {
+///     (
+///         TypedHeader(ContentType::text_utf8()),
+///         "Hello, World!",
+///     )
+/// }
+/// ```
+#[cfg(feature = "headers")]
```

Comment:
I think this should just be `doc(cfg)` since the whole module is already feature-gated like this (or should be).

Reply:
Does that actually make a difference somewhere? I tested it with `RUSTDOCFLAGS="--cfg docsrs" cargo +nightly doc --no-deps` and it looks fine.

Reply:
No I don't think it makes a difference, it's just not so clear why there's a redundant `cfg` attribute there as-is.